05-UI-ReactJS/source/movie.js

```diff
@@ -0,0 +1,27 @@
+import React from "react";
```

Comment:
Movie does not seems to be a good name for this component... please think about a better name.